src/main/webapp/dev/js/main/instructorCourseEnrollPage.js

```diff
@@ -159,11 +159,8 @@ function expandCollapseNewStudentsPanel() {
     const panelHeading = $(this);
     const panelCollapse = panelHeading.parent().children('.panel-collapse');
 
-    if (toggleStudentsPanel(panelHeading, panelCollapse)) { // if panel is shown
-        $('.enroll-students-spreadsheet-buttons').show();
-    } else {
-        $('.enroll-students-spreadsheet-buttons').hide();
-    }
+    toggleStudentsPanel(panelHeading, panelCollapse);
```

Comment:
So the function `toggleStudentsPanel` shouldn't return `true` or `false` anymore. We are not the return value anymore.